### DISCUSSION TITLE
Which NS to deploy External Scaler?

### DISCUSSION THREAD

**Discussion by mishk99** [kedacore/keda#5175] [General]
Wondering which is the best practice: 

to if want to deploy the External Scaler to the 1. KEDA namespace? OR 2. Deployment object namespace? 

The documentation examples show deploying everything into KEDA namespace, but we have many deployments and plan on having different External Scalers for each. so What would be the best practice - to deploy all the External Scalers to KEDA NS? or the diffrent Deployment NS for each service? or somewhere else?

**Comment by tomkerkhove** [comment]
I don't believe there is a best practice and it depends on your organization.


Do you want to re-use external scalers across namespaces/teams? If so, then moving to the namespace with KEDA or a dedicated namespace is best. If not, then keep it simple.
